lucene/test-framework/src/java/org/apache/lucene/analysis/BaseTokenStreamTestCase.java

```diff
@@ -190,7 +196,8 @@ public static void assertTokenStreamContents(TokenStream ts, String[] output, in
       if (posLengthAtt != null) posLengthAtt.setPositionLength(45987653);
       if (keywordAtt != null) keywordAtt.setKeyword((i&1) == 0);
       if (payloadAtt != null) payloadAtt.setPayload(new BytesRef(new byte[] { 0x00, -0x21, 0x12, -0x43, 0x24 }));
-      
+      if (flagsAtt != null) flagsAtt.setFlags(Integer.MAX_VALUE); // all 1's
```

Comment:
that's all bits except the first 1 (`01111111111111111...`). To invert all bits use `-1` or better `~0`

Reply:
oops yeah realized that partway through (when I switched to ~0 for the other file), didn't come back and fix it here.